lib/main.dart

```diff
@@ -1,17 +1,34 @@
+// ignore_for_file: public_member_api_docs
+
+/*
+Copyright 2022 by Mustafa Sezer <mustafa.sezer@hit-solutions.de>
+
+This file is part of picos.
+
+picos is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
```

Comment:
Please format the license notice so that it does not exceed 80 characters per line.